src/pages/ExamsPage/messages.js

```diff
@@ -461,6 +461,46 @@ const messages = defineMessages({
   },
 
   // <AllowanceListActions />
+  editAllowanceHeader: {
+    id: 'AllowanceListActions.editAllowanceHeader',
+    defaultMessage: 'Edit Allowance?',
+    description: 'Header for the edit allowance modal',
+  },
+  editAllowanceBody: {
+    id: 'AllowanceListActions.editAllowanceBody',
+    defaultMessage: 'todo: need to add this',
+    description: 'Body for the edit allowance modal',
+  },
+  editAllowanceUsernameField: {
```

Comment:
Question: some of these fields are the same text as for the add allowance modal. Is it better to have the edit allowance fields to have their own messages (as is), or is it better to reuse existing fields that are the same?

Reply:
I think either is valid. If we were translating, maybe there would be a slight advantage to sharing a single message because it's easier to translate a single message instead of two identical ones. On the other hand, having separate strings does allow for greater flexibility. But I don't think we need that flexibility, and I also think that there's no reason we shouldn't have a single string for something like "Allowance type". I would expect that to be the same between the add and edit modals for the foreseeable future.

If you do want to share messages, I think we should rename the `id`s to be something more generic. For example, `AddAllowanceModal.addAllowanceAllowanceTypeField`  could become `allowanceForm.allowanceTypeLabel` or something.

Reply:
Since the fields are the same, I think it's ok to unify them. Probably renaming those as something like `allowanceUsernameField` as this example and use it in both cases.

Reply:
Sounds good! Updating now. Thanks y'all!